packages/valory/contracts/service_registry/contract.py

```diff
@@ -32,11 +32,13 @@
 EXPECTED_CONTRACT_ADDRESS_BY_CHAIN_ID = {
     1: "0x48b6af7B12C71f09e2fC8aF4855De4Ff54e775cA",
     5: "0x1cEe30D08943EB58EFF84DD1AB44a6ee6FEff63a",
+    137: "0xE3607b00E75f6405248323A9417ff6b39B244b50",
     31337: "0x998abeb3E57409262aE5b751f60747921B33613E",
 }
 DEPLOYED_BYTECODE_MD5_HASH_BY_CHAIN_ID = {
     1: "6f9fc7f3c2348801737120e6b5f8fa8e9670c65152c66d128ff4cddb465b4d705340c559e352f5e7f29bda3b84a8d36d4a9448b791cfe2d370e31c01276e0244",
     5: "d4a860f21f17762c99d93359244b39a878dd5bac9ea6056c0ff29c7558d6653aa0d5962aa819fc9f05f237d068845125cfc37a7fd7761b11c29a709ad5c48157",
+    137: "10e2cfb500481d6c5a3b6b90507e4ac04d8b0d88741cea5568306ed4115f24e8b9747055423da5fca05838d5ccefebf41fb47d2ba1fb45215b6b21c0a27823be",
```

Comment:
Could someone else also double-check the byteode's hash?

Reply:
Verified it using

```python
import web3
import hashlib

my_provider = web3.Web3.HTTPProvider(
    "POLYGON_RPC"
)
w3 = web3.Web3(my_provider)

assert (
"10e2cfb500481d6c5a3b6b90507e4ac04d8b0d88741cea5568306ed4115f24e8b9747055423da5fca05838d5ccefebf41fb47d2ba1fb45215b6b21c0a27823be"
    == hashlib.sha512(
        w3.eth.get_code("0xE3607b00E75f6405248323A9417ff6b39B244b50")
        .hex()
        .encode("utf-8")
    ).hexdigest()
)

```

Reply:
Please double check with Aleks re the code. the l2 code is different so I'm a bit surprirsed it works. Might be because the methods are the same,,

Reply:
@DavidMinarsch it results in a different hash on polygon, as expected.

Reply:
The L2 code re: hashes are exactly the same as on L1